### DISCUSSION TITLE
Get MAC Address?

### DISCUSSION THREAD

**Discussion by duncanam** [siderolabs/talos#8509] [Q&A] [answered]
I'm running Talos through Proxmox on a small home cluster. I'd like to get the mac addresses of each node such that I get set DHCP Lease Reservations on my router for the IP addresses to remain persistent on my network. Is this possible?

**Comment by duncanam** [comment]
Actually, I may have found a solution, through pinging the intranet address then using `arp -a` to get the MAC address. Is there a smarter way to be doing this, however?

**Comment by steverfrancis** [comment, marked as answer]
`taloctl get link`  ?
